be/src/exec/workgroup/scan_task_queue.h

```diff
@@ -18,19 +19,26 @@ struct ScanTask {
     using WorkFunction = std::function<void(int)>;
 
     ScanTask() : ScanTask(nullptr, nullptr) {}
+    explicit ScanTask(WorkFunction work_function) : workgroup(nullptr), work_function(std::move(work_function)) {}
     ScanTask(WorkGroupPtr workgroup, WorkFunction work_function)
             : workgroup(std::move(workgroup)), work_function(std::move(work_function)) {}
     ~ScanTask() = default;
 
-    // Disable copy constructor and assignment.
-    ScanTask(const ScanTask&) = delete;
-    ScanTask& operator=(const ScanTask&) = delete;
+    DISALLOW_COPY(ScanTask);
     // Enable move constructor and assignment.
     ScanTask(ScanTask&&) = default;
     ScanTask& operator=(ScanTask&&) = default;
 
+    bool operator<(const ScanTask& rhs) const { return priority < rhs.priority; }
+    ScanTask& operator++() {
+        priority += 2;
```

Comment:
large priority value means low priority? why +2, should other factors take into consideration, for examples:
1. io cost
2. bytes fetched

Reply:
This is the previous codes, and I just move the position of them.

- High priority value means high priority, because `BlockingPriorityQueue` is a max heap.
- The payload of tasks is `BlockingPriorityQueue`. After taking 512 tasks from the queue, the priority of remaining tasks in the queue will be upgrade 2.

Reply:
> should other factors take into consideration, for examples:
> 
> 1. io cost
> 2. bytes fetched

Sounds reasonable, we could consider some factors, when we optimize the scheduler of scan i/o task. 
This PR doesn't modify the scheduler logic, and all the tasks are FIFO as before, since priorities of all the tasks are still 20.